source/webpack.common.js

```diff
@@ -20,12 +20,17 @@ module.exports = {
 		infrance: ['./source/sites/mycompanyinfrance.fr/entry.js'],
 		embauche: ['./source/sites/embauche.gouv.fr/entry.js'],
 		// To not introduce breaking into the iframe integration, we serve simulateur.js from a 'dist' subdirectory
-		'dist/simulateur': ['./source/sites/embauche.gouv.fr/iframe-script.js']
+		'dist/simulateur': ['./source/sites/embauche.gouv.fr/iframe-script.js'],
+		engine: ['./source/engine/index.js']
```

Comment:
Je pense qu'il vaudrait mieux faire une conf webpack séparée pour la librairie. On risque d'avoir différents besoin (version polyfill, sans polyfill, min). De plus, on ne veut pas forcement la compiler pour le dev, mais uniquement lorsque l'on choisit de la publier.

La solution la plus propre serait de séparer notre repos entre la lib et les sites, chacun avec son propre package.json, la lib étant une dépendance des sites.

Reply:
:+1: 
Pour la séparation des repos, d'accord pour la propreté, mais pas certain de l'efficacité au stade acutel.

Reply:
"Monorepo" et "modulaire" ne sont pas forcément opposés, cf https://medium.com/@maoberlehner/monorepos-in-the-wild-33c6eb246cb9